met-web/src/components/LandingPage/LandingPage.tsx

```diff
@@ -112,13 +112,19 @@ const LandingPage = () => {
             allowSort: false,
             getValue: (row: Engagement) => {
                 if (row.surveys.length === 0) {
-                    return 'N/A';
+                    return '';
                 }
 
                 return (
-                    <MuiLink component={Link} to={`/survey/${Number(row.surveys[0].id)}/results`}>
-                        View Report
-                    </MuiLink>
+                    <>
```

Comment:
The above early retun is there in case we need not to show View Report link. Early return was used because it's cleaner and clearer. Why not use it for when row.surveys[0].responseCount === 0 too